src/kube-runtime/deploy/rbac.yaml

```diff
@@ -18,17 +18,28 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: frameworkbarrier-account
+  name: runtime-account
```

Comment:
why change this name?

Reply:
It's for runtime service, so use such name

---
In reply to: [352481708](https://github.com/microsoft/pai/pull/3944#discussion_r352481708) [](ancestors = 352481708)

Reply:
pls change all occurrences if you want to change name